maven-compat/src/main/java/org/apache/maven/repository/legacy/LegacyRepositorySystem.java

```diff
@@ -602,6 +601,17 @@ private Authentication getAuthentication( RepositorySystemSession session, Artif
                                             authCtx.get( AuthenticationContext.PASSWORD ) );
                     result.setPrivateKey( authCtx.get( AuthenticationContext.PRIVATE_KEY_PATH ) );
                     result.setPassphrase( authCtx.get( AuthenticationContext.PRIVATE_KEY_PASSPHRASE ) );
+                    //MNG-5583 per endpoint PKI authentication
+                    result.setKeyAlias( authCtx.get( "getKeyAlias" ) );
```

Comment:
Can't you re-use the constants you declared in one of the earlier pull requests here?